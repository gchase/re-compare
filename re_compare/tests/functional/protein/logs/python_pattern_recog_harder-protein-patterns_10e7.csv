regex,matchtype,matchnumber,time(ms),span
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),first_match,0,499.3422031402588,"[-1, -1]"
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),all_matches,0,499.3422031402588,"[-1, -1]"
DHKNLD.D,first_match,1,0.9090900421142578,"[613257, 613264]"
DHKNLD.D,consecutive,1,0.9090900421142578,"[613257, 613264]"
DHKNLD.D,consecutive,2,0.003814697265625,"[613397, 613404]"
DHKNLD.D,all_matches,2,0.9129047393798828,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,first_match,1,3.1900405883789062,"[98919, 98925]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,1,3.1900405883789062,"[98919, 98925]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,2,0.013113021850585938,"[99268, 99274]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,all_matches,2,3.203153610229492,"[-1, -1]"
(GI)(G|R)P.Y..K..R,first_match,1,0.5829334259033203,"[214952, 214963]"
(GI)(G|R)P.Y..K..R,consecutive,1,0.5829334259033203,"[214952, 214963]"
(GI)(G|R)P.Y..K..R,consecutive,2,0.0069141387939453125,"[215383, 215394]"
(GI)(G|R)P.Y..K..R,all_matches,2,0.5898475646972656,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),first_match,1,10.131120681762695,"[140636, 140642]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),consecutive,1,10.131120681762695,"[140636, 140642]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),consecutive,2,0.009059906005859375,"[140745, 140751]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),all_matches,2,10.140180587768555,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),first_match,1,5.7849884033203125,"[201600, 201603]"
((N|S.TNH.Y)(F|W)N)(L|I),consecutive,1,5.7849884033203125,"[201600, 201603]"
((N|S.TNH.Y)(F|W)N)(L|I),consecutive,2,1.2443065643310547,"[226966, 226969]"
((N|S.TNH.Y)(F|W)N)(L|I),all_matches,2,7.029294967651367,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,first_match,1,8.95380973815918,"[157517, 157522]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,consecutive,1,8.95380973815918,"[157517, 157522]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,consecutive,2,0.02193450927734375,"[157733, 157738]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,all_matches,2,8.975744247436523,"[-1, -1]"
P..L...KWE.C,first_match,0,18.604040145874023,"[-1, -1]"
P..L...KWE.C,all_matches,0,18.604040145874023,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),first_match,0,333.80985260009766,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),all_matches,0,333.80985260009766,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,first_match,1,4.104852676391602,"[811564, 811573]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,consecutive,1,4.104852676391602,"[811564, 811573]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,consecutive,2,0.0059604644775390625,"[812074, 812083]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,all_matches,2,4.110813140869141,"[-1, -1]"
