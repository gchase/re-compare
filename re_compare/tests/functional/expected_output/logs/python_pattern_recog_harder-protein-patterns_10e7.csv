regex,matchtype,matchnumber,time(ms),span
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),first_match,0,516.3109302520752,"[-1, -1]"
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),all_matches,0,516.3109302520752,"[-1, -1]"
DHKNLD.D,first_match,1,1.2369155883789062,"[613257, 613264]"
DHKNLD.D,consecutive,1,1.2369155883789062,"[613257, 613264]"
DHKNLD.D,consecutive,2,0.0050067901611328125,"[613397, 613404]"
DHKNLD.D,all_matches,2,1.241922378540039,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,first_match,1,4.666805267333984,"[98919, 98925]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,1,4.666805267333984,"[98919, 98925]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,2,0.02002716064453125,"[99268, 99274]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,all_matches,2,4.686832427978516,"[-1, -1]"
(GI)(G|R)P.Y..K..R,first_match,1,0.33593177795410156,"[214952, 214963]"
(GI)(G|R)P.Y..K..R,consecutive,1,0.33593177795410156,"[214952, 214963]"
(GI)(G|R)P.Y..K..R,consecutive,2,0.004291534423828125,"[215383, 215394]"
(GI)(G|R)P.Y..K..R,all_matches,2,0.3402233123779297,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),first_match,1,13.214111328125,"[140636, 140642]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),consecutive,1,13.214111328125,"[140636, 140642]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),consecutive,2,0.017881393432617188,"[140745, 140751]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),all_matches,2,13.231992721557617,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),first_match,1,5.702972412109375,"[201600, 201603]"
((N|S.TNH.Y)(F|W)N)(L|I),consecutive,1,5.702972412109375,"[201600, 201603]"
((N|S.TNH.Y)(F|W)N)(L|I),consecutive,2,0.7607936859130859,"[226966, 226969]"
((N|S.TNH.Y)(F|W)N)(L|I),all_matches,2,6.463766098022461,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,first_match,1,7.01904296875,"[157517, 157522]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,consecutive,1,7.01904296875,"[157517, 157522]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,consecutive,2,0.012159347534179688,"[157733, 157738]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,all_matches,2,7.03120231628418,"[-1, -1]"
P..L...KWE.C,first_match,0,22.699832916259766,"[-1, -1]"
P..L...KWE.C,all_matches,0,22.699832916259766,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),first_match,0,410.7398986816406,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),all_matches,0,410.7398986816406,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,first_match,1,3.7508010864257812,"[811564, 811573]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,consecutive,1,3.7508010864257812,"[811564, 811573]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,consecutive,2,0.012159347534179688,"[812074, 812083]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,all_matches,2,3.762960433959961,"[-1, -1]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),first_match,1,1.8999576568603516,"[41351, 41359]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),consecutive,1,1.8999576568603516,"[41351, 41359]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),all_matches,1,570.8930492401123,"[-1, -1]"
(((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A))(L|I|V|M),first_match,0,19.336938858032227,"[-1, -1]"
(((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A))(L|I|V|M),all_matches,0,19.336938858032227,"[-1, -1]"
(A|G....GK)(S|T),first_match,1,0.006198883056640625,"[57, 58]"
(A|G....GK)(S|T),consecutive,1,0.006198883056640625,"[57, 58]"
(A|G....GK)(S|T),consecutive,2,0.00476837158203125,"[94, 95]"
(A|G....GK)(S|T),all_matches,2,0.010967254638671875,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)GT,first_match,1,50.93884468078613,"[681321, 681326]"
((CP|.)(S|T)N)(I|L|V)GT,consecutive,1,50.93884468078613,"[681321, 681326]"
((CP|.)(S|T)N)(I|L|V)GT,consecutive,2,3.1538009643554688,"[724976, 724982]"
((CP|.)(S|T)N)(I|L|V)GT,all_matches,2,54.0926456451416,"[-1, -1]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,first_match,1,1.6570091247558594,"[413664, 413675]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,consecutive,1,1.6570091247558594,"[413664, 413675]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,consecutive,2,2.7170181274414062,"[1192336, 1192347]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,all_matches,2,4.374027252197266,"[-1, -1]"
